chart/kubeapps/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubeapps
-version: 0.9.5
+version: 0.9.6
```

Comment:
let's separate the chart changes from the code changes, we won't want to release a new chart with this yet

Reply:
also, I'm confused, you've bumped the chart but haven't added the flag?